client/src/pages/automation/project/components/DataPill.tsx

```diff
@@ -29,18 +30,17 @@ const DataPill = ({
     const subProperties = property?.properties || property?.items;
 
     if (!property?.name && (property?.type === 'OBJECT' || property?.type === 'ARRAY')) {
-        property.name = `[${arrayIndex || 0}]`;
+        property.name = '[index]';
```

Comment:
jel ja krivo vidin ili će ovo rezultirati tome da nam piše string `index` u sučelju? Jel to namjerno?

Reply:
@kresimir-coko da, budem ti objasnio zašto :)